corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -325,6 +333,8 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
             'keypress @ui.searchTextBox': 'searchTextKeyAction',
             'keypress @ui.paginationGoTextBox': 'paginationGoKeyAction',
             'keypress @ui.paginators': 'paginateKeyAction',
+            'click @ui.selectAllCheckbox': 'selectAllAction',
```

Comment:
This is going to need a keypress, too, right? We should make a note to do accessibility testing on this as part of QA.

Reply:
Definitely agree-- I'll make a note in the QA details